app/styles/nypr-member-center.scss

```diff
@@ -13,6 +13,8 @@
 @import "nypr-member-center/current-status-detail";
 @import "nypr-member-center/loading-animation";
 
+@import "nypr-account-modal";
+
```

Comment:
The problem I'm running into here is the settings get overwritten by css from `@import "ember-modal-dialog/ember-modal-appearance";` earlier in this file.

Maybe `ember-modal-dialog/ember-modal-appearance` being imported in two places is the issue in the first place. There's definitely a better way to structure this.  Maybe `nypr-account-modal` needs to be  it's own top level export like `nypr-account-settings` and `nypr-member-center`... What do you think?